run.pxi

```diff
@@ -7,7 +7,38 @@
 (require pixie.fs :as fs)
 (require pixie.test :as t)
 
+
+
 (def *all-commands* (atom {}))
+(def unknown-command (atom true))
+(def show-all (atom false))
+(def namespaces (atom 
+  '("" 
+    "pixie.async"
+    "pixie.math"
+    "pixie.stacklets"
+    "pixie.system"
+    "pixie.buffers"
+    "pixie.test"
+    "pixie.channels"
+    "pixie.parser"
+    "pixie.time"
+    "pixie.csp"
+    "pixie.uv"
+    "pixie.repl"
+    "pixie.streams"
+    "pixie.ffi-infer"
+    "pixie.io.common"
+    "pixie.io.tty"
+    "pixie.io.tcp"
+    "pixie.io.uv-common"
+    "pixie.io"
+    "pixie.io-blocking"
+    "pixie.fs"
+    "pixie.set"
+    "pixie.string"
+    "pixie.walk")))
```

Comment:
Can you fold this into one (very long) line?  That way it doesn't take up as much vertical space.